hyperemitter.js

```diff
@@ -26,7 +26,7 @@ function HyperEmitter (db, schema, opts) {
     return new HyperEmitter(db, schema, opts)
   }
 
-  this.messages = protobuf(schema)
+  this.codecs = protobuf(schema)
```

Comment:
Cool, I don't understand the first part though. If which argument is a string? I presume you mean ensure we can add the protobuf schema so we don't break existing peeps? The new sig is slightly different. I'd probably lease schema there then if you are worried about breaking things?